sqlite_fdw.c

```diff
@@ -5157,11 +5160,37 @@ sqlite_to_pg_type(StringInfo str, char *type)
 
 	for (i = 0; pg_type[i][0] != NULL; i++)
 	{
-		if (strncmp(type, pg_type[i][0], strlen(pg_type[i][0])) == 0)
+		const char *t0 = pg_type[i][0];
+		if (strncmp(type, t0, strlen(t0)) == 0)
 		{
+			bool postgis = false;
```

Comment:
I got the following warning for this line, when `make ENABLE_GIS=1`.  
```
sqlite_fdw.c: In function ‘sqlite_to_pg_type’:
sqlite_fdw.c:5166:9: warning: variable ‘postgis’ set but not used [-Wunused-but-set-variable]
    bool postgis = false;
         ^~~~~~~
```
This variable is used in default case only, without SQLITE_FDW_GIS_ENABLE.  
So, you should move this block (from line 5166 to 5177) to that case (under line 5183).

Reply:
Fixed in https://github.com/pgspider/sqlite_fdw/pull/96/commits/feff0255a34cf4b869e8a9895002a57a8516a18a , @son-phamngoc . Please verify.

Reply:
Thanks @mkgrgis . I confirmed. Please help me to resolve this comment.

Reply:
Sorry @mkgrgis . This fix has not been completed.  
At line 5121, `j` variable is only used in default mode without SQLITE_FDW_GIS_ENABLE, so a warning occurred.  
```
sqlite_fdw.c: In function ‘sqlite_to_pg_type’:
sqlite_fdw.c:5121:8: warning: unused variable ‘j’ [-Wunused-variable]
  int   j;
        ^
```
Please fix it.

Reply:
No problems, please verify https://github.com/pgspider/sqlite_fdw/pull/96/commits/f1bdb98b4c93135da3d6094f3853338869e1b72f I have no warnings now.